index.html

```diff
@@ -4,9 +4,74 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <title>🌦 Weather Report 🌦</title>
+    <link rel="stylesheet" href="./styles/index.css">
 </head>
 <body>
-    
+    <header>
+        <h1>Weather Report</h1>
+        <span>
+            For:
+            <span id="headerCityname" class="headerCity_name">
+                Seattle
+            </span>
+        </span>
+    </header>
+    <section class="garden_section" id="garden-section">
+        <h2>Weather Garden</h2>
+        <div class="garden_content">
+            <div id="sky">
+            </div>
+            <div id="landscape">
+                <img src="" id="landscape-img">
+            </div>
+        </div>
+    </section>
+    <section class="city_name_section">
+        <h2>City</h2>
+        <div class="city_content">
+            <label for="cityName"></label>
+            <input type="text" id="cityNameInput" name="name" autocomplete="on" placeholder="Name of City Here">
```

Comment:
Well done keeping this empty to begin with. This allows you to load the state the first time the page renders! You could use the same process on line 52!